tests/test_models.py

```diff
@@ -544,6 +544,18 @@ def test_non_valid_response(self):
         with pytest.raises(ValidationError):
             models.EntityModel(**non_valid_response)
 
+    def test_masking_unchecked_fields(self, mock_boto3):
+        with open("./tests/test_config.yaml", "r") as f:
+            local_config = yaml.safe_load(f)
+
+        if "version" in local_config["product"]:
+            local_config["product"].pop("version")
+        if "eula_document" in local_config["offer"]:
+            local_config["offer"].pop("eula_document")
+
+        entity_model = models.EntityModel.get_entity_from_yaml(local_config)
+        assert entity_model.Description.ProductTitle == "test"
```

Comment:
this needs a more complete validation. we should ensure that all fields that should be stubbed have the expected stub value.